src/pybind/ceph_argparse.py

```diff
@@ -127,7 +127,7 @@ def __init__(self, **kwargs):
         set any per-instance validation parameters here
         from kwargs (fixed string sets, integer ranges, etc)
         """
-        pass
+        self.typeargs = None
```

Comment:
```suggestion
        self.typeargs: Optional[str] = None
```

Reply:
thank you for the suggestion. after revisiting the code, i think i should drop this member variable in a follow-up commit. as it's never assigned.

Reply:
#42043